===================================================================
diff -ruN origin_plugins/cordova-plugin-qrscanner/plugin.xml plugins/cordova-plugin-qrscanner/plugin.xml
--- origin_plugins/cordova-plugin-qrscanner/plugin.xml	null
+++ plugins/cordova-plugin-qrscanner/plugin.xml	null
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <plugin 
-  xmlns="http://www.phonegap.com/ns/plugins/1.0" id="cordova-plugin-qrscanner" version="3.0.1">
+  xmlns="http://www.phonegap.com/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android" id="cordova-plugin-qrscanner" version="3.0.1">
   <name>QRScanner</name>
   <engines>
     <engine name="cordova" version=">=3.4.0"/>
@@ -28,7 +28,6 @@
         <param name="ios-package" value="QRScanner"/>
       </feature>
     </config-file>
-    <dependency id="cordova-plugin-add-swift-support" spec="~1.7.2" />
     <source-file src="src/ios/QRScanner.swift"/>
     <config-file target="*-Info.plist" parent="NSCameraUsageDescription">
       <string>The camera is used to scan QR codes.</string>
===================================================================
diff -ruN origin_plugins/cordova-plugin-qrscanner/src/android/QRScanner.java plugins/cordova-plugin-qrscanner/src/android/QRScanner.java
--- origin_plugins/cordova-plugin-qrscanner/src/android/QRScanner.java	null
+++ plugins/cordova-plugin-qrscanner/src/android/QRScanner.java	null
@@ -24,7 +24,7 @@
 import org.json.JSONObject;
 import android.hardware.Camera;
 import android.provider.Settings;
-import android.support.v4.app.ActivityCompat;
+import androidx.core.app.ActivityCompat;
 import android.view.ViewGroup;
 import android.widget.FrameLayout;
===================================================================
diff -ruN origin_plugins/cordova-plugin-qrscanner/src/android/qrscanner.gradle plugins/cordova-plugin-qrscanner/src/android/qrscanner.gradle
--- origin_plugins/cordova-plugin-qrscanner/src/android/qrscanner.gradle	null
+++ plugins/cordova-plugin-qrscanner/src/android/qrscanner.gradle	null
@@ -4,7 +4,7 @@

 dependencies {
     compile 'com.journeyapps:zxing-android-embedded:3.3.0'
-    compile 'com.android.support:appcompat-v7:23.1.0'
+    compile 'androidx.appcompat:appcompat:1.0.0'
 }

 android {
