--- origin_plugins/cordova-plugin-lottie-splashscreen/src/android/ColorHelper.kt	null
+++ plugins/cordova-plugin-lottie-splashscreen/src/android/ColorHelper.kt	null
@@ -25,6 +25,7 @@
     var color = parseLong(xColorString, 16)

     when {
+        xColorString == "0" -> color = -0x1000000
         xColorString.length == 6 -> color = color or -0x1000000
         xColorString.length != 8 -> throw IllegalArgumentException("Unknown color")
     }
