diff -ruN origin_plugins/cordova-plugin-firebase-lib/src/android/FirebasePlugin.java plugins/cordova-plugin-firebase-lib/src/android/FirebasePlugin.java
--- origin_plugins/cordova-plugin-firebase-lib/src/android/FirebasePlugin.java	null
+++ plugins/cordova-plugin-firebase-lib/src/android/FirebasePlugin.java	null
@@ -1,12 +1,11 @@
 package org.apache.cordova.firebase;

 import android.app.NotificationManager;
-import android.content.ComponentName;
 import android.content.Context;
 import android.content.Intent;
 import android.content.SharedPreferences;
 import android.os.Bundle;
-import android.support.v4.app.NotificationManagerCompat;
+import androidx.core.app.NotificationManagerCompat;
 import android.util.Base64;
 import android.util.Log;

@@ -14,12 +13,11 @@
 import android.app.NotificationChannel;
 import android.content.ContentResolver;
 import android.os.Build;
-import android.support.v4.app.NotificationCompat;
+import androidx.core.app.NotificationCompat;
 import android.media.AudioAttributes;
 import android.net.Uri;

 import com.crashlytics.android.Crashlytics;
-import io.fabric.sdk.android.Fabric;

 import java.lang.reflect.Field;

@@ -56,10 +54,7 @@
 // Firebase PhoneAuth
 import java.util.concurrent.TimeUnit;

-import com.google.firebase.auth.FirebaseAuth;
-import com.google.firebase.auth.AuthResult;
 import com.google.firebase.FirebaseException;
-import com.google.firebase.auth.FirebaseAuthException;
 import com.google.firebase.auth.FirebaseAuthInvalidCredentialsException;
 import com.google.firebase.FirebaseTooManyRequestsException;
 import com.google.firebase.auth.PhoneAuthCredential;
diff -ruN origin_plugins/cordova-plugin-firebase-lib/src/android/FirebasePluginMessagingService.java plugins/cordova-plugin-firebase-lib/src/android/FirebasePluginMessagingService.java
--- origin_plugins/cordova-plugin-firebase-lib/src/android/FirebasePluginMessagingService.java	null
+++ plugins/cordova-plugin-firebase-lib/src/android/FirebasePluginMessagingService.java	null
@@ -8,7 +8,7 @@
 import android.media.RingtoneManager;
 import android.net.Uri;
 import android.os.Bundle;
-import android.support.v4.app.NotificationCompat;
+import androidx.core.app.NotificationCompat;
 import android.util.Log;
 import android.app.Notification;
 import android.text.TextUtils;
